deepinterpolation/generator_collection.py

```diff
@@ -89,6 +96,21 @@ def __get_norm_parameters__(self, idx):
 
         return local_mean, local_std
 
+    def _normalize(self, arr: Union[np.ndarray, tf.Tensor], 
```

Comment:
I understand this function is to avoid duplication in the OphysGenerator. 
At the moment it probably belongs in the OphysGenerator class since it is now used in any other sub-class of DeepGenerator.